### DISCUSSION TITLE
convert: CorruptImageProfile `xmp' @ warning/profile.c/SetImageProfileInternal/1882.

### DISCUSSION THREAD

**Discussion by hailiangchen** [ImageMagick/ImageMagick#1982] [General]
[root@bogon ~]# /usr/local/ImageMagick/bin/convert -version
Version: ImageMagick 7.0.10-10 Q16 x86_64 2020-05-05 https://imagemagick.org
Copyright: © 1999-2020 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenMP(3.1) 
Delegates (built-in): bzlib freetype jng jpeg ltdl png tiff x zlib
[root@bogon ~]# /usr/local/ImageMagick/bin/convert 6372235628325592129805091.jpg 1.png
convert: CorruptImageProfile `xmp' @ warning/profile.c/SetImageProfileInternal/1882.

Why does the above tip appear?I'm compiling and installing, am I missing any dependencies?

**Comment by dlemstra** [comment]
This is a warning (not error) that your input image contains an invalid xmp profile.

**Reply by hailiangchen** [reply]
Ok, thanks

**Comment by fmw42** [comment]
try adding -quiet to your command right after magick. For IM 7, use magick, not convert.
